e2e_tests/test_download.py

```diff
@@ -0,0 +1,9 @@
+from playwright.sync_api import Page
+
+
+def test_has_title(authenticated_page: Page):
```

Comment:
can we rename this to test_download_record

and rename the page to test_record.py
?